package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "vscode_deno",
+  "name": "deno",
```

Comment:
revert

Reply:
I was not able to get the extension working locally with `name` set to `vscode_deno`, possibly because of hardcoded value in [extension.ts](https://github.com/denoland/vscode_deno/blob/1a6b9060ffe9a0d5d6ba77dfb3e9399c88f5aa06/client/src/extension.ts#L23) 
```typescript
const denoExtensionId = "denoland.deno";
```

same goes for `publisher` set to `the deno authors`